Doc/library/asyncio-graph.rst

```diff
@@ -0,0 +1,131 @@
+.. currentmodule:: asyncio
+
+
+.. _asyncio-graph:
+
+===================
+Stack Introspection
+===================
+
+**Source code:** :source:`Lib/asyncio/graph.py`
+
+-------------------------------------
+
+asyncio has powerful runtime call graph introspection utilities
+to trace the entire call graph of a running coroutine or task, or
+a suspended *future*.  These utilities and the underlying machinery
+can be used by users in their Python code or by external profilers
+and debuggers.
+
+.. versionadded:: 3.14
+
+
+.. function:: print_call_graph(*, future=None, file=None, depth=1)
+
+   Print the async call graph for the current task or the provided
+   :class:`Task` or :class:`Future`.
+
+   The function receives an optional keyword-only *future* argument.
+   If not passed, the current running task will be used. If there's no
+   current task, the function returns ``None``.
+
+   If the function is called on *the current task*, the optional
+   keyword-only *depth* argument can be used to skip the specified
+   number of frames from top of the stack.
+
+   If *file* is not specified the function will print to :data:`sys.stdout`.
+
+   **Example:**
+
+   The following Python code:
+
+   .. code-block:: python
+
+      import asyncio
+
+      async def test():
+         asyncio.print_call_graph()
+
+      async def main():
+         async with asyncio.TaskGroup() as g:
+            g.create_task(test())
```

Comment:
To ease C/C, can you use 4-spaces indent here? (you still need a 3 space indent to align with `.. code-block`).

Reply:
Absolutely, no idea how I ended up putting 3 spaces there :) Fixing.